include/threads/ThreadState.h

```diff
@@ -10,6 +11,20 @@
 
 namespace threads {
 
+class SerialisedLevel
+{
+  public:
+    int32_t depth;
+    int32_t effectiveDepth;
+    int32_t maxActiveLevels;
+    int32_t nThreads;
+
+    uint32_t nSharedVars = 0;
+    uint32_t* sharedVars;
+};
```

Comment:
The `SerializedLevel` is the context that gets sent with remote batch OpenMP requests